package.json

```diff
@@ -39,6 +39,7 @@
     "@rhoas/app-services-ui-shared": "0.16.6",
     "@rhoas/smart-events-management-sdk": "0.51.0",
     "@xstate/react": "3.0.1",
+    "date-fns": "2.29.3",
```

Comment:
General question about stories.
Are all dependencies used in them considered dev or prod dependencies?

Reply:
@vpellegrino I would say dev. All storybook deps are `devDependencies`.

`date-fns` is an exception though. It was already [used](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/d95c69d18a3834dea9e4b5aba28ff2c7b10a44fe/src/app/Instance/InstanceDetails/InstanceDetails.tsx#L22) [elsewhere](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/d95c69d18a3834dea9e4b5aba28ff2c7b10a44fe/src/app/components/SEStatusLabel/useResourceDelayedStatus.ts#L6) as transitive dep. 
When importing it inside a story I've finally made things right.

Reply:
Makes sense. Thank you @kelvah